keyboards/kprepublic/bm40hsrgb/keymaps/dan/rules.mk

```diff
@@ -1,2 +1,4 @@
-TAP_DANCE_ENABLE = yes
 SRC += features/custom_shift_keys.c
+TAP_DANCE_ENABLE = no
+IGNORE_MOD_TAP_INTERRUPT = yes
+HOLD_ON_OTHER_KEY_PRESS = no
```

Comment:
These are not `rules.mk` settings, so specifying them here will do absolutely nothing.
```suggestion
```